newIDE/app/src/InstancesEditor/InstancesMover.js

```diff
@@ -107,12 +105,15 @@ export default class InstancesMover {
         this.options.gridOffsetY,
         this.options.gridType
       );
-      roundedTotalDeltaX = magnetPosition[0] - initialMagnetX;
-      roundedTotalDeltaY = magnetPosition[1] - initialMagnetY;
     } else {
-      roundedTotalDeltaX = this.totalDeltaX;
-      roundedTotalDeltaY = this.totalDeltaY;
+      // Without a grid, it's rounded to the nearest pixel
+      // the size might not be round
+      // so the magnet corner is still relevent
```

Comment:
Nitpicking: Name what is "it", the size of what, and be sure to delimitate sentences:

// Without a grid, the position is still rounded to the nearest pixel.
// The size of the instance (or selection of instances) might not be round,
// so the magnet corner is still relevant.